src/index.html

```diff
@@ -22,6 +22,291 @@
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review_count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price_text">Price:</p>
+          <p class="card__price_count">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review_count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price_text">Price:</p>
+          <p class="card__price_count">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review_count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price_text">Price:</p>
+          <p class="card__price_count">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The product card elements are repeated multiple times with identical content. Consider using a templating engine or JavaScript to dynamically generate these cards to avoid redundancy and improve maintainability.